layouts/partials/head.html

```diff
@@ -18,8 +18,7 @@
 {{ hugo.Generator }}
 
 <!--Favicon-->
-<link rel="shortcut icon" href="{{ site.Params.favicon | absURL }}" type="image/x-icon">
-<link rel="icon" href="{{ site.Params.favicon | absURL }}" type="image/x-icon">
+{{ partial "favicon.html" . }}

```

Comment:
I think the best implementation would be to keep the two original lines in the partial within this project, then override it in the site repo by placing it in `/layouts/partials/favicon.html`

That way the theme doesn't refer to files which don't exist in the theme

Reply:
Ok, I also need to move `head.html` I'll do this